### PR TITLE
#157648309 set up eslint

### PR DESCRIPTION
**Pull request by OKiMaureen**
#### What does this PR do?
-Set up eslint
 #### Description of Task to be completed?
-Eslint will use airbnb guide
#### What are the relevant pivotal tracker stories?
-#157648309
 #### Questions:
none
#### Screenshots:
none

### CODE REVIEW

File: eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "root": true,
+    "extends": "airbnb-base",
```
Comment (paradisekelechi) [inline eslintrc.json]: @OKiMaureen  You are to use `airbnb` and not `airbnb-base`

File: eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "env": {
+      "node": true,
+      "es6": true,
+      "mocha": true
+    },
+    "rules": {
```
Comment (paradisekelechi) [inline eslintrc.json]: Where are all these rules coming from?

Comment (OKiMaureen) [reply eslintrc.json]: I was sent a script at the last boot-camp with this rules

File: eslintrc.json (hunk 1 above)
Comment (OKiMaureen) [reply eslintrc.json]: okay

File: eslintrc.json (hunk 2 above)
Comment (OKiMaureen) [reply eslintrc.json]: should i remove the rules?

Comment (paradisekelechi) [reply eslintrc.json]: Remove the rules. Kindly check the product specification and do as it says. Check it very well

Comment (OKiMaureen) [reply eslintrc.json]: ok

**Pull request merged**